fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -66,27 +69,26 @@ const EventRollCall = (props: IPropTypes) => {
   };
 
   const onReopenRollCall = () => {
-    if (!event.idAlias) {
+    if (eventHasBeenOpened) {
+      requestReopenRollCall(event.idAlias).catch((e) => {
+        makeToastErr('Unable to send Roll call re-open request');
+        console.debug('Unable to send Roll call re-open request', e);
+      });
+    } else {
       makeToastErr('Unable to send roll call re-open request, the event does not have an idAlias');
       console.debug('Unable to send roll call re-open request, the event does not have an idAlias');
```

Comment:
if the strings are repeated identically, you can also consider putting them in a variable